lib/widgets/write_comment.dart

```diff
@@ -113,15 +113,9 @@ class WriteComment extends HookWidget {
           child: Stack(
             children: [
               ListView(
```

Comment:
Does this need to be a ListView? It has only 3 children.
I feel a `Column` would be more performant, and would let the user just scroll the preview widget.

Reply:
Yeah good call. Especially now that I took out the other scroll widget. Will implement.

Reply:
Columns aren't scrollable. List view is required